client/blocks/reader-post-card/style.scss

```diff
@@ -159,14 +159,26 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	fill: lighten( $gray, 10% );
 }
 
-.reader-post-card .gravatar {
+.reader-post-card .reader-avatar .gravatar,
+.reader-post-card .reader-avatar .site-icon {
 	float: left;
 	height: 32px;
 	margin: 2px 6px 0 0;
 	vertical-align: text-top;
 	width: 32px;
 }
 
+.reader-post-card .reader-avatar .site-icon {
+	width: 32px !important;
```

Comment:
fixed in https://github.com/Automattic/wp-calypso/pull/8904/commits/12e705bb9ecd3fa3ab273c7f8be3af335f5af2fc